### DISCUSSION TITLE
Integration Error

### DISCUSSION THREAD

**Discussion by diogodogbr** [djansen1987/SAJeSolar#13] [Q&A] [answered]
Good night,

I installed the integration with SAJ, my inverter is the R5-4K-S2 and it is giving me this error

![Screenshot_7](https://user-images.githubusercontent.com/59851199/137235055-17e2ee5e-a613-485a-9b27-2dd000687e80.jpg)

Can you help me
(Sorry about my English)

**Comment by djansen1987** [comment, marked as answer]
Could you share your config (without username/password)

**Reply by diogodogbr** [reply]
I sent it on your email github-h1@djansen.nl

**Reply by djansen1987** [reply]
Solved
